docs/my-website/docs/observability/keywordsai_integration.md

```diff
@@ -0,0 +1,73 @@
+# 🗝️ Keywords AI - LLM monitoring platform
+
+:::tip
+
+This is community maintained. Please make an issue if you run into a bug:
+https://github.com/BerriAI/litellm
+
+:::
+
+[Keywords AI](https://keywordsai.co/) makes it easy for developers to build LLM applications. With 2 lines of code, developers get a complete monitoring platform that speeds up deploying & monitoring AI apps in production.
+
+## Using Keywords AI with LiteLLM
+
+LiteLLM provides `callbacks`, allowing you to easily log data to Keywords AI based on the status of your responses.
+
+### Supported LLM Providers
+
+Keywords AI can log requests across [various LLM providers](https://docs.keywordsai.co/integration/supported-models)
```

Comment:
I'm confused, why does your logging integration not work across all litellm models? 

You don't handle any of the calling logic - it should just be a destination to receive the logged request/response (similar to otel/langfuse/etc.)

Reply:
I will change the wording on this. Yes it works across litellm models

Reply:
Changed